pagopa/pn-frontend
# updated payment attachment api - pn-7488

## Short description
Updated the api that downloads the attachments for a payment

## List of changes proposed in this pull request
- Updated notification.routes.ts
- Updated Notifications.api.ts
- Updated actions.ts
- Updated the NotificationPaymentPagoPa.tsx

## How to test
Log in with grossini (dev environment) -> go to the detail of TJUN-ATLX-UNQN-202307-L-1 -> try to download the payment file and check that the api is called correctly
